src/ui/components/shared/Modals/SourcemapSetupModal.tsx

```diff
@@ -0,0 +1,38 @@
+import classNames from "classnames";
+import React from "react";
+import { connect, ConnectedProps } from "react-redux";
+import * as actions from "ui/actions/app";
+import { PrimaryButton } from "../Button";
+import { Dialog, DialogActions, DialogDescription, DialogLogo, DialogTitle } from "../Dialog";
+import Modal from "../NewModal";
+
+function SourcemapSetupModal({ hideModal }: PropsFromRedux) {
+  const onClick = () => {
+    window.open("https://replayio.notion.site/Adding-Source-Maps-1923e679c1e4411db1bda29536eb1e31");
+  };
+
+  return (
+    <Modal options={{ maskTransparency: "translucent" }} onMaskClick={hideModal}>
+      <Dialog
+        className={classNames("flex flex-col items-center")}
+        style={{ animation: "dropdownFadeIn ease 200ms", width: 400 }}
+      >
+        <DialogLogo />
+        <DialogTitle>Replay is better with sourcemaps</DialogTitle>
+        <DialogDescription>
+          {`We noticed that you have sourcemaps, but they haven't been uploaded to our servers yet.
+          Ready to get started?`}
```

Comment:
### MSG 1, if they have NextJS

We noticed that you're using NextJS which makes adding sourcemaps super easy. 

https://docs.replay.io/docs/1923e679c1e4411db1bda29536eb1e31#6e444abdd19642af9ddc34766ff84bf2

### MSG 2, if they have a sourceURL

We noticed that you have sourcemaps, but they are not publicly available. Would you like to upload them to Replay?

https://docs.replay.io/docs/1923e679c1e4411db1bda29536eb1e31#912af13ef09a41f4ae774a90796ebbc1